lib/models/finamp_models.dart

```diff
@@ -110,6 +126,19 @@ class FinampSettings {
     this.autoloadLastQueueOnStartup = _autoLoadLastQueueOnStartup,
     this.hasCompletedBlurhashImageMigration = true,
     this.hasCompletedBlurhashImageMigrationIdFix = true,
+    this.hasCompleteddownloadsServiceMigration = true,
+    this.requireWifiForDownloads = false,
+    this.onlyShowFullyDownloaded = false,
+    this.showDownloadsWithUnknownLibrary = true,
+    this.maxConcurrentDownloads = 10,
+    this.downloadWorkers = 5,
+    this.resyncOnStartup = false,
```

Comment:
I think we can make the default true, it's a very useful feature and most people will probably expect this to happen anyway.  

Also, have you thought about scheduling periodic syncs? Unlesss I'm reinstalling the app, Finamp never gets closed on my phone, because I'm almost constantly listening to music. I think the foreground thread is shut down after a while, but the notification always stays active. I'm not sure if that's enough to trigger a resync when I go back into the app or resume playing, and if not we might need periodic syncs in addition to the startup sync.

Reply:
I haven't really thought about that, my phone kills finamp all the time.  If the background player is still alive, I don't think that re-triggers the startup code that does the resync.  I've changed the default.

Reply:
Could you reuse the `AppLifecycleListener` for that? And check if the app has been in the background longer than x hours, and if yes start a sync?

Reply:
Okay, I did that.  It should now also try to resync if in background for >5 hours.